src/components/Walk/ResultsPage.js

```diff
@@ -2,22 +2,132 @@ import React from "react";
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import Grid from '@mui/material/Grid';
+import { BarChart, Bar, LabelList, Cell, XAxis, YAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer } from 'recharts';
+
+/*
+Expected API data format for both host and Participant results page.
+{
+    barDefaultColor: str,  // (required)    - default color of the bar graphs
+    data: [ // data is a sorted array. The left most item in the array will contain data for the left most bar chart 
+        {
+            barName: str,  (required) - This will be displayed at the bottom of a bar. Can be an empty string.
+            color: str,    (not required) - Specific color of a single bar graph.
+            count: number,  (required)  - The height of the bar graph.
+            participantLocation: boolean (required) - This shows the location of the participant. If sending data to host this will always be false
+        },
+        {
+            ....
+        }
+    ]
+}
+
+See example below
```

Comment:
Image of example below:

![image](https://user-images.githubusercontent.com/25392726/163686245-dbdcfe1d-4878-4375-abf2-03944fb9e33e.png)